### DISCUSSION TITLE
How can I get in to the Slack

### DISCUSSION THREAD

**Discussion by DamianMac** [zephyrproject-rtos/zephyr#36636] [Q&A] [answered]
Hey there.
I'm new to Zephyr and figuring my way, I keep seeing a link to get a slack invite, but it only lets me sign up if I have an email domain on the special list. Is the Slack open to the public? If so, how does one ask for an invite?

Thanks so much.

**Comment by erwango** [comment]
You'll find slack invite link here: https://zephyrproject.org/contact-us/

**Reply by DamianMac** [reply]
The invite link takes me here though.

![image](https://user-images.githubusercontent.com/1187840/124043148-c822e880-da4d-11eb-9378-1e60288d847b.png)

and I don't work for any of those companies.

Or are you saying I should send an email on the contact page?

**Reply by erwango** [reply]
This shouldn't be required.

@kestewart could help on this maybe ?

**Comment by stephanosio** [comment, marked as answer]
FYI, updated Slack invite link here:
https://tinyurl.com/2vue8666

Fixed here: https://github.com/zephyrproject-rtos/zephyr/pull/36669

**Comment by DamianMac** [comment]
Awesome thank you so much!
